dev-infrastructure/templates/svc-cluster.bicep

```diff
@@ -29,8 +29,13 @@ param disableLocalAuth bool
 @description('Deploy ARO HCP RP Azure Cosmos DB if true')
 param deployFrontendCosmos bool
 
+<<<<<<< HEAD
```

Comment:
Looks like some rebase artifacts that just need some adjusting here

Reply:
fixed. thank you!